src/app/routes/resetpwd/resetpwd.component.html

```diff
@@ -97,8 +97,106 @@ <h1>You have requested to reset your password</h1>
             </td>
         </tr>
     </table>
+    </div>
     <!--/100% body table-->
+    <div class="d-block d-md-none rwdcontainer reset">
+      <table
+      style="@import url(https://fonts.googleapis.com/css?family=Rubik:300,400,500,700|Open+Sans:300,400,600,700); font-family: 'Open Sans', sans-serif;">
+<tr>
+    <td>
+        <table width="100%" align="center">
+            <tr>
+                <td style="height:80px;">&nbsp;</td>
+            </tr>
+            <tr>
+                <td>
+                  
+                  
+                </td>
+            </tr>
+            <tr>
+                <td style="height:20px;">&nbsp;</td>
+            </tr>
+            <tr>
+                <td>
+                    <table class="table">
+                        <tr>
+                            <td style="height:40px;">&nbsp;</td>
```

Comment:
do not do nbsp, remove from everywhere and use padding, margin, min-width or min-height